Parser/Client/Browser.php

```diff
@@ -235,6 +235,7 @@ class Browser extends ClientParserAbstract
         'SU' => 'Super Fast Browser',
         'S3' => 'surf',
         'S0' => 'START Internet Browser',
+        'S4' => 'Steam',
```

Comment:
Wondering if we maybe should name it `Steam Overlay` or `Steam GameOverlay` instead

Reply:
Long time since I played video games. Also, we could name it `Steam In-Game Overlay`. If so, I should also change it for `Origin`.

Reply:
`Steam In-Game Overlay` sounds fine